src/monitoring_tests/fee_test.py

```diff
@@ -0,0 +1,160 @@
+"""
+Fee Test
+"""
+
+from src.monitoring_tests.base_test import BaseTest
+from src.apis.web3api import Web3API
+from src.apis.orderbookapi import OrderbookAPI
+from src.helper_functions import get_logger
+from src.constants import (
+    FEE_ABSOLUTE_DEVIATION_ETH_FLAG,
+    FEE_RELATIVE_DEVIATION_FLAG,
+)
+
+
+class FeeTest(BaseTest):
+    """
+    Class for testing fees.
+    """
+
+    def __init__(self):
+        self.web3_api = Web3API()
+        self.orderbook_api = OrderbookAPI()
+        self.logger = get_logger()
+
+    def run(self, tx_hash) -> bool:  # pylint: disable=too-many-locals
```

Comment:
Before even reading the function implementation, ignoring too many locals give me the impression that the function itself could be broken down into simpler "helper functions". If that is not the case, then you could also consider creating an object whose fields are some of the values you are declaring here.

Reply:
I tried splitting it. We could even split `FeeTest` it into two independent tests `CostCoverage` and `FeeQuoteTest` or something since they are essentially independent tests.